projects/go-lib/src/lib/components/go-table/go-table-config.model.ts

```diff
@@ -2,7 +2,7 @@ import { GoTablePageConfig } from './go-table-paging.model';
 import { GoTableSortConfig } from './go-table-sort.model';
 import { GoTableSearchConfig } from './go-table-search-config.model';
 
-export class GoTableConfig {
+export class GoTableConfig<T = any> {
```

Comment:
Do we need `= any`? Or we could simplify it to `<T>`?

Reply:
@jiaweicai92 yes we do need this. If we don't set `T` to a type then there are errors in the `go-table.component.ts` file because it needs a type set. Additionally, this allows us to ship this without any breaking changes.

Reply:
Thanks for explaining this 🥇